ExperimentRunner/PluginHandler.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, name, params):
         self.paths = paths.paths_dict()
 
         self.plugin_base = PluginBase(package='ExperimentRunner.plugins')
-        if self.name_lower == 'android' or self.name_lower == 'trepn' or self.name_lower == 'batterystats':
+        if self.name_lower in ['android', 'trepn', 'batterystats', 'frametimes', 'garbagecollection']:
```

Comment:
@EricZielinski Is it possible to avoid to have to hardcode all the names of the plugins here? The base code of Android Runner should be fully plugin-independent.

Reply:
I adjusted it to the same approach used for plugins in a test folder's Plugins package while taking your request of having sub-directories into account. 

Adding an AndroidRunner default plugin now requires you to make a sub-directory with the Plugin file inside (MetricX -> `metricx/MetricX.py`).